dags/veda_data_pipeline/utils/cogify_transfer/requirements.txt

```
@@ -1,7 +1,7 @@
 aws-lambda-powertools
 awslambdaric
 boto3
-pystac==1.4.0
+pystac==1.14.1
```

```suggestion
pystac
```
we probably don't even need to pin the version here (we are not pinning in other locations)